seed/php-sdk/imdb/src/Core/RawClient.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace Seed\Core;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Utils;
+use InvalidArgumentException;
+use Psr\Http\Client\ClientExceptionInterface;
+use Psr\Http\Message\StreamInterface;
+use Seed\ClientOptions;
+
+
+class RawClient
+{
+    /**
+     * The client options used to make requests.
+     *
+     * @var ClientOptions
+     */
+    private ClientOptions $clientOptions;
+
+    /**
+     * The client options used to make requests.
+     *
+     * * @param ClientOptions|null $clientOptions
+     */
+    public function __construct(
+        ?ClientOptions $clientOptions,
+    )
+    {
+        $this->clientOptions = $clientOptions ?? new ClientOptions();
+    }
+
+    /**
+     * The client options used to make requests.
+     *
+     * @throws ClientExceptionInterface
+     */
+    public function sendRequest(
+        BaseApiRequest $request,
+    ): BaseApiResponse
+    {
+        $httpRequest = $this->buildRequest($request);
+        $httpResponse = $this->clientOptions->httpClient->send($httpRequest);
+        return new BaseApiResponse(
+            $httpResponse->getStatusCode(),
+            $httpResponse,
```

Comment:
what's the purpose of pulling out the status code here if they can get it from the `httpResponse` itself?

Reply:
Primarily for convenience + consistency with the C# generator ([link](https://github.com/fern-api/fern/blob/d6ccbc91c2a7ffea9a1deba62912ee5175c17538/seed/csharp-sdk/imdb/no-custom-config/src/SeedApi/Core/RawClient.cs#L72)).

Given that that this is entirely internal I'm also fine with just returning the underlying `ResponseInterface` for now. Might as well keep things as simple as we can until there's a more compelling reason for it

Reply:
Sweet, will drop `BaseApiResponse` in favor of `ResponseInterface` 👍